.circleci/config.yml

```diff
@@ -613,18 +634,41 @@ commands:
             java -version
             ./mvnw -v
             python3 -V
-      - run:
-          name: run integration test
-          command: |  # Runs normally if it is not being run in parallel
-            export DOCKSTORE_CACHE_MIGRATIONS=true
-            if [ $CIRCLE_NODE_TOTAL != 1 ] 
-            then
-              ./mvnw -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate -P$TESTING_PROFILE,coverage -ntp \
-              clean verify -Dit.test=`cat /tmp/tests-to-run.txt | tr ' ' ','` -DfailIfNoTests=false | grep  -v "^Running Changeset:"
-            else 
-              ./mvnw -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate clean install -P$TESTING_PROFILE,coverage -ntp | grep  -v "^Running Changeset:"
-            fi
-          # The piping grep command is a temporary fix to this issue https://github.com/liquibase/liquibase/issues/2396
+      - when:
+          condition: <<parameters.use_hoverfly_cert>>
+          steps:
+            - run:
+                name: run integration test with hoverfly cert
+                command: |                  
+                  # Adding all "normal" certs into this local one that has the Hoverfly cert (instead of adding Hoverfly cert to the global one so it doesn't potentially affect other tests)
```

Comment:
You're probably on this already, but I think you may be able to avoid repetition with a script or with more steps (without the condition)